ui/src/test/java/edu/wpi/grip/ui/codegeneration/tools/JavaPipelineInterfacer.java

```diff
@@ -0,0 +1,124 @@
+    try {
+      pipeline.getMethod("setsource" + num, value.getClass()).invoke(instance, value);
+    } catch (NoSuchMethodException | SecurityException | IllegalAccessException
+        | IllegalArgumentException | InvocationTargetException e) {
+      e.printStackTrace();
+      fail("setsource" + num + " is not valid for class " + value.getClass().getSimpleName() + ""
+          + " because there was a: ");
+    }
+  }
+
+  /* (non-Javadoc)
+   * @see edu.wpi.grip.ui.codegeneration.tools.PipelineInterfacer#setSourceAsObject(int, java
+   * .lang.Object)
+   */
+  @Override
+  public void setSourceAsObject(int num, Object value) {
+    try {
+      pipeline.getMethod("setsource" + num, Object.class).invoke(instance, value);
+    } catch (NoSuchMethodException | SecurityException | IllegalAccessException
+        | IllegalArgumentException | InvocationTargetException e) {
+      e.printStackTrace();
+      fail("setsource" + num + " is not valid for class " + value.getClass().getSimpleName() + ""
+          + " because there was a: ");
+    }
+  }
+
+  /* (non-Javadoc)
+   * @see edu.wpi.grip.ui.codegeneration.tools.PipelineInterfacer#setMatSource(int, java.io.File)
+   */
+  @Override
+  public void setMatSource(int num, File img) {
+    setSource(num, Imgcodecs.imread(img.getAbsolutePath()));
+  }
+
+  /* (non-Javadoc)
+   * @see edu.wpi.grip.ui.codegeneration.tools.PipelineInterfacer#process()
+   */
+  @Override
+  public void process() {
+    try {
+      pipeline.getMethod("process").invoke(instance);
+    } catch (IllegalAccessException | IllegalArgumentException
+        | InvocationTargetException | NoSuchMethodException
+        | SecurityException e) {
+      e.printStackTrace();
+      fail("process method doesn't exist");
+    }
+  }
+
+  /* (non-Javadoc)
+   * @see edu.wpi.grip.ui.codegeneration.tools.PipelineInterfacer#getOutput(int)
+   */
+  @Override
+  public Object getOutput(int num, GenType type) {
+    try {
+      return pipeline.getMethod("getoutput" + num).invoke(instance);
+    } catch (IllegalAccessException | IllegalArgumentException
+        | InvocationTargetException | NoSuchMethodException
+        | SecurityException e) {
+      e.printStackTrace();
+      fail("getoutput" + num + " method does not exist");
+      return null;
+    }
+  }
+
+  /* (non-Javadoc)
+   * @see edu.wpi.grip.ui.codegeneration.tools.PipelineInterfacer#setSwitch(int, boolean)
+   */
+  @Override
+  public void setSwitch(int num, boolean value) {
+    try {
+      pipeline.getMethod("setSwitch" + num, boolean.class).invoke(instance, value);
+    } catch (NoSuchMethodException | SecurityException | IllegalAccessException
+        | IllegalArgumentException | InvocationTargetException e) {
+      e.printStackTrace();
+      fail("setSwitch" + num + "is not a valid method");
+    }
+  }
+
+  /* (non-Javadoc)
+   * @see edu.wpi.grip.ui.codegeneration.tools.PipelineInterfacer#setValve(int, boolean)
+   */
```

Comment:
This is an Overridden method. Javadoc is not needed -- it is automatically added from the super class.